src/base_classes/state_info.cc

```diff
@@ -37,6 +37,13 @@ namespace forte {
 StateInfo::StateInfo(int na, int nb, int multiplicity, int twice_ms, int irrep)
     : na_(na), nb_(nb), multiplicity_(multiplicity), twice_ms_(twice_ms), irrep_(irrep) {}
 
+const std::vector<std::string> StateInfo::irrep_labels = {"","","","","","","",""};
```

Comment:
@lcyyork: as a quick bug fix I removed the call to `Molecule->...` because it was giving a segmentation fault. I think it is an order of initialization issue.